src/engine/sparqlExpressions/UuidExpressions.h

```diff
@@ -70,4 +71,10 @@ class UuidExpression : public SparqlExpression {
   std::span<SparqlExpression::Ptr> childrenImpl() override { return {}; }
 };
 
+}  //  namespace detail::uuidExpression
+
+using namespace detail::uuidExpression;
```

Comment:
Sonarcloud is right.
The `using namespace` here completely defeats the purpose (everything inside `detail:uuidExpression` becomes part of `sparqlExpression`.
Instead you have to explicitly use the verbose forms in the following two lines
`using UuidExpr = Impl<detail::uuid::fromIri, detail::uuid::irirUUidKey>` etc.